var/spack/repos/builtin/packages/nvhpc/package.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import os
+
+
+class Nvhpc(Package):
+    """The NVIDIA HPC Software Development Kit (SDK) includes the proven compilers,
+       libraries and software tools essential to maximizing developer productivity
+       and the performance and portability of HPC applications.
```

Comment:
`remark` : as this is NVIDIA's hpc-sdk, I was wondering if this should be available to users as `nv-hpc-sdk` or `nvidia-hpc-sdk`.

Reply:
its name changed to `nvidia-hpc-sdk`